### PR TITLE
[upd] - Issue #7 - Rework functions.php structure

### PR DESCRIPTION
**Pull request by FeetloveR**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -1,100 +1,3 @@
...
-	public function add_to_twig( $twig ) {
-		/* this is where you can add your own fuctions to twig */
-		return $twig;
-	}
-
-}
-
-new StarterSite();
+/* Everything is defined in the lib folder */
+require_once('lib/autoload.php');
```
Comment (Plou) [inline functions.php]: :+1:

**Comment by Plou** [comment]
:+1: 

I think some files could be merged back together like menu/main and menu/footer, to ease integration. 

Great work, i'm ok to merge asap.

File: functions.php (hunk 1 above)
Comment (toofff) [inline functions.php]: :+1:

**Comment by toofff** [comment]
Great work!

**Pull request merged**